### DISCUSSION TITLE
Pseudo before and after element implementation in core?

### DISCUSSION THREAD

**Discussion by jshimkoski** [tailwindlabs/tailwindcss#3981] [Ideas]
I created a plug-in that supports the new JIT implementation that creates before and after variants as well as pseudo-content-{value} utility classes.

Would this be something you guys would like in Tailwind CSS core?

**Comment by jshimkoski** [comment]
Repo available here: https://github.com/shimyshack/tailwindcss-pseudo-element-plugin

You can install this via: `npm install @shimyshack/tailwindcss-pseudo-element-plugin`

**Comment by olets** [comment]
See Adam Wathan's response at https://github.com/tailwindlabs/tailwindcss/discussions/2119#discussioncomment-272395
